### ISSUE TITLE
A section inside chat, All Shared Media / All Uploads

### ISSUE THREAD

**Issue by orbitneon** [simplex-chat/simplex-chat#1899]
Signal alike

Example:
![2bac7031-f31c-48b7-8275-9fe38c9f2f99](https://user-images.githubusercontent.com/54729767/216840452-7ef1920d-81b2-4cdf-b8c0-f1702a91cf7f.png)

![386b81a6-55a6-4a64-8025-50e7a1e332ec](https://user-images.githubusercontent.com/54729767/216840469-2e8acef1-5265-4c00-9a88-637504526228.png)

**Comment by Dituar** [comment]
Better Photos and videos, Files, Audio, Links.  And when viewing/selecting this photo and video, file, audio, link, you could go to this message in the chat.

**Comment by Samylol99** [comment]
Yes that'd improve the experience a lot. I wonder if this is planned.

**Comment by Narasimha-sc** [comment]
> Yes that'd improve the experience a lot. I wonder if this is planned.

yes, it is planned
